video5_classandselectors/class5.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="style5.css">
+    <title>css basics </title>
+</head>
+<body>
+
+    <div id="content">
+        <p class="success feedback">
+            aaaaaaaaaaaaafbshfsnfgsngsng8n8net8gneignifjgndfgndifjgnifng
```

Comment:
use keyword lorem for dummy texts. For eg., if you need a text with 30 characters then use the keyword lorem30 and enter.